src/templates/layouts/healthCareLocalFacility/OperatingStatus.tsx

```diff
@@ -10,14 +14,9 @@ import { HealthCareLocalFacility as FormattedHealthCareLocalFacility } from '@/t
  */
 export const OperatingStatusFlags = ({
   operatingStatusFacility,
-  menu,
-}: Pick<
-  FormattedHealthCareLocalFacility,
-  'operatingStatusFacility' | 'menu'
->) => {
-  const linkBasePath = menu.data.links[0].url.path
-
-  if (!linkBasePath || operatingStatusFacility === 'normal') {
+  basePath,
+}: OperatingStatusFlagsProps) => {
+  if (!basePath || operatingStatusFacility === 'normal') {
```

Comment:
I changed this component because I'm going to start using it in my template. That probably means it should move out of this folder, but I didn't think we were ready for that. I actually had more code related to the usage of this, but I accidentally clobbered it beyond recovery, like a noob.